### PR TITLE
Style form om

### PR DESCRIPTION
**Pull request by Atos20**
#### What's this PR do?  

- it styles `<Form>` component 
- it adds functionality to remove a user's choice when the same choice is clicked again
- it adjusts styling in `<Homepage>` and `<BroswerPage/>`

#### Where should the reviewer start? 

- `<Form>`
- `<Homepage>`
-  `<BroswerPage/>`
#### How should this be manually tested?  
run `npm start`

#### Screenshots?

Screen Shot 2020-11-03 at 9 29 13 AM

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 import { BoardGame } from '../BoardGame/BoardGame';
 import { CardGame } from '../CardGame/CardGame';
-import React, {Component} from 'react';
-import {Route} from 'react-router-dom';
-import { getAllMovies, getAllPodcasts, getAllCardGames, getAllMusic, getAllBoardGames} from '../../apiCalls.js'
-import { BrowsePage } from '../BrowsePage/BrowsePage'
-import { Footer } from '../Footer/Footer'
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { getAllMovies,
```
Comment (nicolegooden) [inline src/Components/App/App.js:12]: Thanks for making this more readable!

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -7,14 +7,22 @@ import { FaServer, FaGithub} from "react-icons/fa";
...
 
     return (
-        <>
+        
           <footer className="social-section">
             <ul className="footer-links">
-              <li className="foot-link"><a target="_blank" href="https://github.com/Atos20/indecision-maker-server"><FaServer className="repo-link"/></a></li>
-              <li className="foot-link"><a target="_blank" href="https://github.com/nicolegooden/indecision-maker"><FaGithub className="repo-link"/></a></li>
+              <li className="foot-link">
+                <a target="_blank" href="https://github.com/Atos20/indecision-maker-server">
```
Comment (nicolegooden) [inline src/Components/Footer/Footer.js:14]: I've seen errors in the console for using `target="_blank"` , wondering if there is a different attribute or value for this attribute that we can utilize.

File: src/Components/Form/Form.js [hunk 3]
```diff
@@ -16,6 +18,10 @@ export class Form extends Component {
   updateCurrentAnswers = (event) => {
     event.preventDefault();
     if (this.state.currentAnswers.includes(event.target.textContent)){
+      const answersPerQuestion = [...this.state.currentAnswers]
```
Comment (nicolegooden) [inline src/Components/Form/Form.js:21]: Very sound logic here.  Thank you for making this so clean and easy to understand.

**Review by nicolegooden — APPROVED** [review]
Orlando, thank you so much for handling so much of our design.  You have done a tremendous job.

File: src/Components/Footer/Footer.js (hunk 2 above)
Comment (Atos20) [reply src/Components/Footer/Footer.js:14]: You might be right, I will double-check that.

File: src/Components/App/App.js [hunk 4]
```diff
@@ -76,8 +83,19 @@ class App extends Component {
...
         </Route>
 
+        <Route 
+          exact 
+          path='/form'>
+            <Form 
+            activities={this.state.activities} 
+            setActivities={this.setActivities}
+            updateActivityAnswers={this.updateActivityAnswers}
+            />
```
Comment (BlakeDonn) [inline src/Components/App/App.js:93]: My branch has `determineRandomActivities` being passed as a prop here, just a note for if we run into conflicts

File: src/Components/Form/Form.js [hunk 5]
```diff
@@ -44,17 +50,21 @@ export class Form extends Component {
...
+                <div className='choice'>
+                  <h3
+                    key={i}
+                    id={data[index].answerType}
+                    onClick={this.updateCurrentAnswers} 
+                    value={choice} 
+                    className='option'>{choice}
+                  </h3>
+                </div>
+                )
```
Comment (BlakeDonn) [inline src/Components/Form/Form.js:67]: We will also probably run into conflicts here regarding the anonymous function on my end, we will have to talk about what makes more sense.

**Review by BlakeDonn — APPROVED** [review]
Styling looks great! We may have a few conflicts as noted but we can tackle those together!

**Pull request merged**